server/db/driver/pg/internal/matches.go

```diff
@@ -0,0 +1,43 @@
+package internal
+
+const (
+	CreateMatchesTable = `CREATE TABLE IF NOT EXISTS %s (
+		matchid BYTEA PRIMARY KEY,
+		takerOrder BYTEA,      -- INDEX this
+		takerAccount BYTEA,    -- INDEX this
+		makerOrder BYTEA,      -- INDEX this
+		makerAccount BYTEA,    -- INDEX this
+		quantity INT8,
+		rate INT8,
+		status INT2,
+		sigTakerMatch BYTEA,
+		sigMakerMatch BYTEA,
+		sigTakerAudit BYTEA,
+		sigMakerAudit BYTEA,
+		sigTakerRedeem BYTEA,
+		sigMakerRedeem BYTEA
+	)`
```

Comment:
I feel like it makes sense to include an epoch identifier too.  However, epoch index is not unambiguous since epoch duration changes it's meaning.  An `eidx:dur` representation of the epoch ID would be unique however.  Needs thought...

Reply:
It is necessary if we want to reconstruct market history from storage and validate matching. I suppose that is something we should at least have the ability to do.